sandbox-web/src/main/java/com/exadel/sandbox/team5/OrderRestController.java

```diff
@@ -0,0 +1,71 @@
+package com.exadel.sandbox.team5;
+
+import com.exadel.sandbox.team5.dto.OrderDto;
+import com.exadel.sandbox.team5.mapper.MapperConverter;
+import com.exadel.sandbox.team5.service.OrderService;
+import com.exadel.sandbox.team5.util.OrderCriteria;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/orders")
+@RequiredArgsConstructor
+public class OrderRestController {
+
+    private final OrderService orderService;
+    private final MapperConverter mapper;
+
+    @GetMapping("/{id}")
+    public OrderDto getOrder(@PathVariable Long id){
+        return orderService.getById(id);
+    }
+
+    @GetMapping("/all")
```

Comment:
It is better to use `api/orders` instead of `api/orders/all`

Reply:
done